### ISSUE TITLE
Ward of Laze follows summoner

### ISSUE THREAD

**Issue by wetbrownsauce** [azerothcore/azerothcore-wotlk#1093]
##### SMALL DESCRIPTION:
Ward of Laze summoned by Mosh'Ogg Witch Doctors in Stranglethorn Vale are capable of movement despite being totems. 

##### EXPECTED BLIZZLIKE BEHAVIOUR:
Ward of Laze should remain stationary always. 

##### CURRENT BEHAVIOUR:
Ward of Laze ( https://wowgaming.altervista.org/aowow/?npc=2667 ) moves around and follows the Witch doctor ( https://wowgaming.altervista.org/aowow/?npc=1144 ) who summoned it. 

##### STEPS TO REPRODUCE THE PROBLEM:
1. Go to the Mosh'Ogg ogres just southeast of Grom'Gol Base Camp in Stranglethorn Vale. 
2. Aggro a Mosh'Ogg Witch Doctor.
3. When it summons its Ward of Laze, run away and watch the totem follow along. 

##### BRANCH(ES):
master

##### AC HASH/COMMIT:
50aace0

##### OPERATING SYSTEM:
Windows 10

##### OTHER CUSTOMIZATIONS:
My only customizations are client-side addons for UI.

**Comment by poszer** [comment]
Confirmed, as Blizzlike, when summoned as Guardian, Ward of Laze should always stay in one position.

**Comment by poszer** [comment]
> 
> 
> try this
> 
> ```
> UPDATE creature SET MovementType = 0 WHERE id = 2667; -- this is useless cause it's already to 0
> UPDATE creature_template SET InhabitType = 8 WHERE entry = 2667; -- this roots the creature
> ```
> 
> and TC removed that column and we still haven't gotten backups of the wiki to expand upon it @Sdyess :(

Nothing, still the same >.<

**Comment by BarbzYHOOL** [comment]
o_0 how possible, are you sure? did you reload the server?

**Comment by poszer** [comment]
> 
> 
> o_0 how possible, are you sure? did you reload the server?

Yeah. The creature has already movement to stay in position, but still moving. Also changing inhabit to 8 did nothing xD

**Comment by BarbzYHOOL** [comment]
Then when the creature is summoned, it has `MovementType` set to something else than 0 (but it's not in the DB since it's summoned by a spell)

**Comment by poszer** [comment]
Ok, this can be as temp fix for it. It will stop the Guardian to follow owner
```

UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 2667;

DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 2667);
INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
(2667, 0, 0, 0, 54, 0, 100, 0, 0, 0, 0, 0, 18, 4, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Ward of Laze - On Just Summoned - Set Unit Flags - Disable Movement'),
(2667, 0, 1, 0, 0, 0, 100, 0, 3, 3, 3, 3, 11, 3826, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Ward of Laze - In Combat - Cast \'Ward of Laze Passive\'');
```

**Comment by poszer** [comment]
I have tested it, and it's fine now (Guardian doesn't follow it anymore) . Will open PR for this 
![wardoflaze2](https://user-images.githubusercontent.com/41213210/50422862-9c0bc880-084e-11e9-9a69-f63a32bb5461.png)
![ward of laze](https://user-images.githubusercontent.com/41213210/50422863-9c0bc880-084e-11e9-9e08-5aed01eee4c0.png)

**Comment by BarbzYHOOL** [comment]
Ok nice
